### ISSUE TITLE
consider incorporating metaformats into core mf2 parsing spec

### ISSUE THREAD

**Issue by snarfed** [microformats/microformats2-parsing#73]
Should we? Or not? I have no clue, deferring to people who understand standards and mf2 parsing better.

https://microformats.org/wiki/metaformats

At least two implementations so far:
* https://github.com/microformats/microformats-parser/pull/229
* https://github.com/microformats/mf2py/pull/213

Earlier related projects:
* https://www.jvt.me/posts/2022/03/14/opengraph-mf2/
* https://david.shanske.com/2022/03/14/meta-tags-to-microformats/

cc @tantek @aciccarello @kevinmarks @barnabywalters @jackyalcine @capjamesg

**Comment by aciccarello** [comment]
I think the spec needs to be refined some more at a minimum before considering incorporating it into the core spec. With the challenge of different sites using meta tags inconsistently, we'll probably need to do more experimentation to find a good balance of correctness and usability.

### Other thoughts:
- I haven't contributed much to specs before so I can't speak to the work of managing standards
- My use case for metaformats revolves mostly around link previews so keeping it as a separate fallback feels reasonable to me.
- I suspect some real-world applications might need some additional translation to better handle some one-off patterns on larger sites but that's probably outside of even the metaformats spec.

**Comment by sknebel** [comment]
agreed with @aciccarello , it needs more refinement, practical testing and checks that adding it won't interfere with existing uses before we should consider adding it to the main spec, if at all. Personally, I think it makes a lot of sense for it to stay separate entirely.

**Comment by snarfed** [comment]
More concurring opinions that this might not belong in the core spec or parsers: https://github.com/microformats/mf2py/pull/213#issuecomment-1831670791

> I think it would make sense for metaformats to be an external extension at this point, since I kind of expect that people would want/need to experiment with variations of it instead of relying on it being updated with mf2py.
